region/crateria/central.json

```diff
@@ -4985,6 +4982,48 @@
                   ],
                   "note": "Quickly grapple to then release grapple on the grapple blocks a few times until they stop working. Be careful not to fall into the spikes.",
                   "devNote": "Grapple cannot be used to get over the spikes, as the PLMs are overloaded too quickly."
+                },
+	        {
+                  "name": "G-Mode Morph Overload PLMs by PBing Super Item",
+                  "notable": false,
+                  "requires": [
+		    {"itemNotCollectedAtNode": 3},
+		    "canRiskPermanentLossOfAccess",
+		    {"ammo": { "type": "PowerBomb", "count": 4 }},
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "direct",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      {"and": [
+                        "SpringBall",
+                        {"spikeHits": 3},
```

Comment:
These spike hits are going to need a crystal flash but I don't that is to be handled here.